pkgs/by-name/ts/tslib/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  clangStdenv,
+  fetchFromGitHub,
+  cmake,
+}:
+
+clangStdenv.mkDerivation (finalAttrs: {
+  pname = "tslib";
+  version = "1.23";
+
+  src = fetchFromGitHub {
+    owner = "libts";
+    repo = finalAttrs.pname;
+    rev = finalAttrs.version;
+    sha256 = "sha256-2YJDADh/WCksAEIjngAdji98YGmwjpvxSBZkxAwFc7k=";
+  };
```

Comment:
shouldn't `hash` also be override-able by `finalAttrs`?

Reply:
How do you mean ? Could you share the change that you are suggesting.

Reply:
Well then I suppose I would have to defne sha256 outside src and then use it as a finalAttrs but I think that would just be unnecessary.

Reply:
As far as i know the `finalAttrs` pattern is meant to make overridden values propagate, but if this is done the package would break as the source hash would have to be overridden with `overrideAttrs`. I think that is a footgun for users.

Reply:
Ah okay, I see your point.